testsuite/MDAnalysisTests/analysis/test_psa.py

```diff
@@ -36,7 +36,8 @@ def setUp(self):
         self.universe_rev = MDAnalysis.Universe(PSF, DCD)
         self.universes = [self.universe1, self.universe2, self.universe_rev]
         self.psa = MDAnalysis.analysis.psa.PSAnalysis(self.universes,           \
-                                               path_select='name CA')
+                                               path_select='name CA',           \
+                                               targetdir=tempdir.TempDir())
```

Comment:
Looking at the rest of the code, I think your only option is to do things explicitly with 
```python
import tempfile
import shutil

# ....
    def setUp(self):
          self.tempdir = tempfile.mkdtemp()
          ...
          self.psa =  MDAnalysis.analysis.psa.PSAnalysis(self.universes, ..., targetdir=self.tempdir)

    def tearDown(self):
            if self.tempdir:
                 shutil.rmtree(self.tempdir)
            self.name = None
```